README.md

```diff
@@ -2,58 +2,63 @@
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- Major languages: HTML
+- Frameworks: Visual Studio
+- Technologies used: Visual Studio, HTML, Linter Code check, bash.
 
 ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://leonardoeng13.github.io/review-trials2/)
 
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+In the previous lessons, you got an introduction to the code review process and you upgraded your HTML and CSS skills. That means that you are ready to perform a code review on your code!
 
+IMPORTANT NOTE: Read all requirements before you start building your project..
 
-To get a local copy up and running follow these simple example steps.
+### Project Requirements 
 
-### Prerequisites
+- Create a new empty repo.
+- Copy your code from the project that you completed during the Admissions Trial: - If you do not have the code from the Admissions Trial ask someone from your Standup Team to share their code with you. In that case, remember to add an attribution note and acknowledgment in your README file.
+- Create a new branch for your copied code.
+- Open a pull request.
+- Play the role of the Code Reviewer:
+    - Follow the list from the article "How to give a code review?".
+    - Check if you have used semantic HTML tags.
+    - Check if you applied HTML & CSS best practices.
+- Add as many comments as possible in the given time and finish your review. Remember to add a minimum of 2 comments - you will definitely find something that could have been done better!
+- Copy a link to the pull request and ask for a code review in your Student Dashboard as usual. (The only difference will be that the Code Reviewer won't check your code but the review you performed - isn't it a code review inception? ;))
 
 ### Setup
 
+- Don't forget to create a workflow repo and install the linter
+
 ### Install
 
-### Usage
+- Follow the instruction as before.
 
 ### Run tests
 
-### Deployment
+- Commit all and copile linter
 
 
 
 ## Authors
 
 👤 **Author1**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: [@Leonardoeng13](https://github.com/leonardoeng13)
+- Twitter: [@Leo de Andrade](https://twitter.com/andrade_leo)
+- LinkedIn: [Leonardo de Andrade](https://linkedin.com/in/leonardodeandrade)
```

Comment:
It's good to know you made the changes in solo mode, but when the Microverse requests an exercise with a code partner, don't suppress who worked with you. Remember to give the rights credits to who helped/developed with you.